ec2/ec2-create-instance.sh

```diff
@@ -8,15 +8,17 @@ BRANCH_DEFAULT='develop'
 PEM_DEFAULT=${HOME}
 
 usage() {
-  echo "Usage: $0 -b <branch> -r <repo> -p <pem_dir> -g <group_vars> -a <dataverse-ansible branch> -t tag" 1>&2
+  echo "Usage: $0 -b <branch> -r <repo> -p <pem_dir> -g <group_vars> -a <dataverse-ansible branch> -i aws_image -s aws_size -t aws_tag" 1>&2
   echo "default branch is develop"
   echo "default repo is https://github.com/IQSS/dataverse"
   echo "default .pem location is ${HOME}"
   echo "example group_vars may be retrieved from https://raw.githubusercontent.com/IQSS/dataverse-ansible/master/defaults/main.yml"
+  echo "default AWS AMI ID is ami-9887c6e7"
```

Comment:
Instead of hard coding "ami-9887c6e7" here maybe we could create a variable called `DEFAULT_AWS_IMAGE` and use it here and in the "else" below. That way we only have it hard coded once.